src/packages/@ncigdc/modern_components/environment.js

```diff
@@ -80,34 +90,34 @@ const fetchQuery = addControlledAccessParams => (operation, variables, cacheConf
         }
 
         return json;
-      })
-      .catch(err => {
-        if (err.status) {
-          switch (err.status) {
-            case 401:
-            case 403:
-              consoleDebug(err.statusText);
-              if (IS_AUTH_PORTAL) {
-                return redirectToLogin('timeout');
-              }
-              break;
-            case 400:
-            case 404:
-              consoleDebug(err.statusText);
-              break;
-            default:
-              return consoleDebug(`Default error case: ${err.statusText}`);
-          }
-        } else {
-          consoleDebug(
-            `Something went wrong in environment, but no error status: ${err}`,
-          );
+      }))
+    .catch(err => {
```

Comment:
Relocated error handling, which had been applied to the response parsing, instead of the query itself.